src/annotator/util/frame-util.js

```diff
@@ -43,6 +43,16 @@ function isValid (iframe) {
   return iframe.className !== 'h-sidebar-iframe';
 }
 
+function isReady (iframe, callback) {
```

Comment:
I see what you mean, in that case `isDocumentReady` works with me. I'll update the PR with this.
I was originally going to call this `isDOMReady` or `isDomReady` but the choice of case threw me off.